pages/levels/level4/index.js

```diff
@@ -0,0 +1,377 @@
+import {useEffect, useState} from "react";
+import Canvas from "../../../components/Canvas";
+import {dataLevel4} from "../../../components/LevelData/_dataLevel4";
+import Ressources from "../../../components/Ressources";
+import styled from "styled-components";
+import OverlayBig from "../../../components/OverlayBig";
+import Header from "../../../components/Header";
+import Cards from "../../../components/Cards";
+import {allCardsData} from "../../../components/LevelData/_allCardsData";
+import OverlaySmall from "../../../components/OverlaySmall";
+import {cardsDeckData} from "../../../components/LevelData/_cardsDeckData";
+import CardsSvg from "../../../components/SVG/CardsSvg";
+import QuestionMarkSvg from "../../../components/SVG/QuestionMarkSvg";
+
+export default function Level4() {
+  const [array, setArray] = useState(dataLevel4.fields);
+  const [chooseTileState, setChooseTileState] = useState(false);
+  const [overlayState, setOverlayState] = useState("tutorial");
+  const [textState, setTextState] = useState(1);
+  const [counter, setCounter] = useState(0);
+  const [allCards, setAllCards] = useState(allCardsData);
+  const [cardsDeck, setCardsDeck] = useState(
+    cardsDeckData.map(index => allCards[index])
+  );
+  const [cardToAdd, setCardToAdd] = useState(false);
+  const [shuffledCards, setShuffledCards] = useState([1]);
+  const [randomCards, setRandomCards] = useState([]);
+  const [chosenCard, setChosenCard] = useState(false);
+  const [activeCard, setActiveCard] = useState("");
+  const [gatherRessources, setGatherRessources] = useState(false);
+  // -------------------ressources--------------------------------------------------
+  const [activeBuildings, setActiveBuildings] = useState(1);
+  const [wood, setWood] = useState(10);
+  const [stone, setStone] = useState(10);
+  const [food, setFood] = useState(10);
+  const [workers, setWorkers] = useState(2);
+  const [dailyWorkers, setDailyWorkers] = useState(2);
+  //this is just for deployment:
+  if (stone === 10000000) {
+    setAllCards(allCards + 1);
+    setActiveBuildings(activeBuildings + 1);
+    setWorkers(0);
+    setTextState(0);
+    setChooseTileState(chooseTileState + 1);
+  }
+  //---------------Winning----------------------------------
+  // if (array.filter(item => item.color === "windmill").length === 1) {
+  //   setTimeout(() => setOverlayState("win"), 1500);
+  // }
```

Comment:
Please delete this unused code.

Reply:
That was exactly the code i forgot to comment in again :D